albatross.opam

```diff
@@ -11,6 +11,8 @@ depends: [
   "dune"
   "dune-configurator"
   "conf-pkg-config" {build}
+  "conf-libnl3" {os = "linux"}
+  "conf-linux-libc-dev" {os = "linux"}
```

Comment:
I haven't tried but I think https://github.com/roburio/albatross/pull/58 might be enough to not require `conf-linux-libc-dev`.

Reply:
thanks @kit-ty-kate